### ISSUE TITLE
i trained the project by my data, but got a model different with your model.

### ISSUE THREAD

**Issue by GentlemanWally** [stevewongv/SPANet#8]
i trained the project by my data, but got a model different with your model.
the below pictures,first is my model,second is your model
![图片](https://user-images.githubusercontent.com/26771177/61270659-e8136000-a7d4-11e9-8afd-7096204c3b8a.png)
![图片](https://user-images.githubusercontent.com/26771177/61270726-0f6a2d00-a7d5-11e9-8c53-e25168d66d3f.png)

**Comment by GentlemanWally** [comment]
when i use the model of mine, the next error is appear. how could i to deal with the problem?
![图片](https://user-images.githubusercontent.com/26771177/61270894-9919fa80-a7d5-11e9-87aa-7539d1aaeb63.png)

**Comment by stevewongv** [comment]
A little different in model size is possible. Second, please check the multi GPU option in main.py

**Comment by GentlemanWally** [comment]
@stevewongv i think you should change the code in line 118 ( **self.net.load_state_dict(obj['net'])** ) of main.py to the code **self.net.load_state_dict({k.replace('module.',''):v for k,v in torch.load(ckp_path)['net'].items()})** .the method can solve the above problem and load model when runing test code.

**Comment by GentlemanWally** [comment]
i want to know how many steps of your latest and how many steps i can test the rain picture significant change?

**Comment by stevewongv** [comment]
> @stevewongv i think you should change the code in line 118 ( **self.net.load_state_dict(obj['net'])** ) of main.py to the code **self.net.load_state_dict({k.replace('module.',''):v for k,v in torch.load(ckp_path)['net'].items()})** .the method can solve the above problem and load model when runing test code.

👌🏻 Thanks for your suggestion. I will fix later.

**Comment by stevewongv** [comment]
> i want to know how many steps of your latest and how many steps i can test the rain picture significant change?

The new version of latest is 40000 steps which is used to evaluate in our test set and real data.

**Comment by stevewongv** [comment]
> i want to know how many steps of your latest and how many steps i can test the rain picture significant change?

I suggest use our latest file to evaluate the real data.

**Comment by GentlemanWally** [comment]
thank you for your work and your advice
